fintech-examples/fintech-ui/src/app/common/navbar/navbar.component.ts

```diff
@@ -29,7 +30,7 @@ export class NavbarComponent implements OnInit {
   getSessionValidUntil(): string {
     const validUntilDate: Date = this.storageService.getValidUntilDate();
     if (validUntilDate != null) {
-      const validUntilDateString = validUntilDate.toLocaleString();
+      const validUntilDateString = Consts.toLocaleString(validUntilDate);
```

Comment:
The entire regex-part of function can be simplified to:
```
var time = date.toLocaleTimeString('en-GB')
```

Reply:
Yes, but I have several places where this is used. I definitly want to have DateFormating handled in one place. Specially, toLoclaTimeString has a seconde Parameter, we do not use yet.

Reply:
You can create utility function and use it everywhere

Reply:
I thought, thats what I did. I put unitilty funktion in consts.ts.